go/sql/builder.go

```diff
@@ -275,7 +275,7 @@ func BuildUniqueKeyRangeEndPreparedQueryViaOffset(databaseName, tableName string
 
 	uniqueKeyColumnNames := duplicateNames(uniqueKeyColumns.Names())
 	uniqueKeyColumnAscending := make([]string, len(uniqueKeyColumnNames))
-	uniqueKeyColumnDescending := make([]string, len(uniqueKeyColumnNames))
+	uniqueKeyColumnDescending := make([]string, len(uniqueKeyColumnNames)) // TODO unused variable
```

Comment:
outdated comment?

Reply:
Ah, nope! This method only uses ascending order, just removed it in [74f60a9](https://github.com/github/gh-ost/pull/1500/commits/74f60a97c6d3afb588a80cc0a58c2e2fed37f6b4)